src/implementations/twig/storybook/package.json

```diff
@@ -7,7 +7,7 @@
   "description": "Storybook EC Twig",
   "scripts": {
     "build": "build-storybook -c .storybook -o build",
-    "start": "start-storybook -p 6006 -s ../../../presets/ec-core/dist,../../../presets/eu-core/dist -c .storybook",
+    "start": "start-storybook -p 6006 -s ../../../presets/ec-core/dist,../../../presets/eu-core/dist,../../../presets/ec-preset-editor/dist -c .storybook",
```

Comment:
the status of this editor preset is still unclear to me, as it is redondent with the concept of default html styling (see https://github.com/ec-europa/europa-component-library/pull/1707)
We can have it as is for now, but we still need to see how to deal with it

Reply:
Removed in order to avoid future confusions until there is a plan for it